mlflow/store/tracking/file_store.py

```diff
@@ -337,7 +337,14 @@ def _create_experiment_with_id(self, name, experiment_id, artifact_uri, tags):
         )
```

Comment:
Can we also update `sqlalchemy_store.py`?

Reply:
I was thinking this would be handled since we have set sqlalchemy defaults.

Reply:
Does `SqlAlchemyStore.restore_experiment` automatically update the last_update_time column? I don't it does.

Reply:
I will check and add test cases for the same. I will make this as draft PR.

Reply:
you are right. I have added them.